_data/utilities.yml

```diff
@@ -506,16 +506,12 @@ websites:
 
   - name: Verizon
     url: https://www.verizon.com/
-    twitter: VerizonSupport
-    facebook: verizon
-    img: verizon.png
-    regions:
-      - us
-
-  - name: Verizon Wireless
-    url: https://www.verizonwireless.com/
-    twitter: VZWSupport
     img: verizon.png
+    tfa:
+      - sms
+      - email
```

Comment:
Where about is email 2FA mentioned on the page? I can only see references to the "My Verizon app" and SMS